packages/squiggle-lang/src/rescript/Distributions/PointSetDist/AlgebraicShapeCombination.res

```diff
@@ -118,13 +118,14 @@ let combineShapesContinuousContinuous = (
   | #Logarithm => (m1, m2) => log(m1) /. log(m2)
   } // note: here, mInv2 = mean(1 / t2) ~= 1 / mean(t2)
 
-  // TODO: I don't know what the variances are for exponentatiation
+  // TODO: I don't know what the variances are for exponentatiation or logarithms
```

Comment:
shouldn't this `TODO` be "the exponentiation and logarithm cases are currently wrong and totally made up" if there are dummy implementations there? or is the whole `TODO` stale?

Reply:
Fair enough